src/Controller/Place.php

```diff
@@ -8,6 +8,19 @@
 
 class Place extends AbstractController
 {
+    public function index(Request $request)
+    {
+        $placeRepo = $this->getDoctrine()->getRepository(\App\Entity\Place::class);
+        $places = $placeRepo->findAll();
```

Comment:
What if we have 10000+ places?
Data should be loaded only if needed, like the index page

Reply:
Fixed